include/deal.II/grid/grid_tools_cache.h

```diff
@@ -138,6 +138,16 @@ namespace GridTools
                 typename Triangulation<dim, spacedim>::active_cell_iterator>> &
     get_cell_bounding_boxes_rtree() const;
 
+    /**
+     * Return the cached RTree object of bounging boxes containing locally owned
+     * active cells, constructed using the active cell iterators of the stored
+     * triangulation.
```

Comment:
I think "locally owned" implies "active". I had to read through the description of this and the previous function several times before I really understood the difference. Would you mind adding a sentence to both functions that contrasts what they do? E.g., "In contrast to the previous function, this function builds the RTree using only the locally owned cells, but does not include ghost cells."

Reply:
What we really want is "non artificial" and "locally owned" but not ghost. How can we call these?

Reply:
I've updated the documentation.